python/python/lance/vector.py

```diff
@@ -202,17 +205,14 @@ def compute_partitions(
     """Compute partitions using GPU kmeans."""
     import torch
 
-    from lance.torch.async_dataset import async_dataset
     from lance.torch.data import LanceDataset as PytorchLanceDataset
 
-    torch_ds = partial(
-        PytorchLanceDataset,
+    with PytorchLanceDataset(
```

Comment:
I dont think we need the `with ... as` any more

Reply:
Fixed